docs/development_environment.md

```diff
@@ -24,11 +24,15 @@ The easiest way to get started with development is to use Visual Studio Code wit
 **Getting started:**
 
 1. Fork the repository.
-2. Clone the repository to your computer.
+2. Clone the repository to your computer*.
 3. Open the repository using Visual Studio code.
 
 When you open this repository with Visual Studio code you are asked to "Reopen in Container", this will start the build of the container.
 
+\****Optimizing performance for Windows***
+
+See more info at '[Open a WSL 2 folder in a container on Windows](https://code.visualstudio.com/docs/remote/containers#_open-a-wsl-2-folder-in-a-container-on-windows)'. Placing your files within the WSL file system will increase the performance of your devcontainer significantly.
```

Comment:
Either we should use note/information blocks as we use everywhere else on this website, or, I think this can just be part of the instruction steps above.

I think the latter is better, making it a part of the instructions makes it friendlier to read.

Reply:
I have place the comment inline now